modules/webgl/src/adapter/resources/webgl-vertex-array.ts

```diff
@@ -97,16 +97,16 @@ export class WEBGLVertexArray extends VertexArray {
       // Attaches ARRAY_BUFFER with specified buffer format to location
       this.device.gl.vertexAttribPointer(location, size, type, normalized, stride, offset);
     }
-    // Clear binding - keeping it may cause [.WebGL-0x12804417100]
-    // GL_INVALID_OPERATION: A transform feedback buffer that would be written to is also bound to a non-transform-feedback target
-    this.device.gl.bindBuffer(GL.ARRAY_BUFFER, null);
```

Comment:
@ibgreen probably this had the same root cause, and we can remove it now?  Unit tests are passing and I double-checked the attribute-transitions demo in deckgl against the change, let me know if there's anything else you know to test for this issue.

Reply:
Wow - was it just a stupid bug all along?

Reply:
The invalid operation error is back with the removal of this :(